R/calcCapacity.R

```diff
@@ -11,75 +11,64 @@
 calcCapacity <- function(subtype) {
 
   if ((subtype == "capacityByTech_windoff") || (subtype == "capacityByTech")) {
+    description <- "Historical capacity by technology." # now always includes offshore wind
+
+    # Use IRENA data for world renewables capacity.
+    # Year: 2000-2017
+    # Technologies: "csp", "geohdr", "hydro", "spv", "windon", "windoff"
+    mapping_IRENA <- tribble(
```

Comment:
use camel case (this is not enforced, but encouraged in mrremind), i.e `mappingIRENA`.